Bytes split function (#7917)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/scalar-functions.md

SPLIT(col1, delimiter)
```

Splits a string into an array of substrings or

> bytes into an array of subarrays based

Reviewer: ```suggestion
bytes into an array of subarrays, based
```

on a delimiter. If the delimiter is not found,

> then the original string or bytes is returned as the only

Reviewer: ```suggestion
the original string or byte array is returned as the only
```

element in the array. If the delimiter is empty,

> then all characters in the string or bytes in the bytes value are split.

Reviewer: ```suggestion
every character in the string or byte in the array is split.
```

> If either, string/bytes or delimiter, are NULL, then a

Reviewer: ```suggestion
Returns NULL if either parameter is NULL.
```

> NULL value is returned.

Reviewer: ```suggestion

```

If the delimiter is found at the beginning or end

> of the string or bytes, or there are contiguous delimiters,

Reviewer: ```suggestion
of the string or byte array, or there are contiguous delimiters,
```
